### DISCUSSION TITLE
Incorrect markdown rendering with <details> tag

### DISCUSSION THREAD

**Discussion by worthant** [community/community#64713] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

## As you see in the screenshot below, link anchors crosses the border, which is not really cool:
![изображение](https://github.com/community/community/assets/43885024/a1cabf4f-72ef-44bf-8892-efa94727ff78)

Check by yourself: https://github.com/Imtjl

Here is how that code looks: 
![изображение](https://github.com/community/community/assets/43885024/3b28d647-ebb1-454e-bf2c-e19dbd1c09a8)

**Comment by ShonChassel** [comment]
Try to give some style

<head>
    <style>
        /* Add styles to adjust the positioning of anchor links */
        a.anchor-link {
            margin-left: 20px; /* Adjust this value as needed */
        }
    </style>
</head>

**Reply by worthant** [reply]
If i remember correctly github only supports centering, and don't support any styles at all
Tried writing inline styles already, didn't work

Here i tried and nothing changed:
|![изображение](https://github.com/community/community/assets/43885024/0f5cdfb6-28b5-467d-b010-7254da439667)|![изображение](https://github.com/community/community/assets/43885024/a510ed6e-98f5-4256-8415-5a0f27b0da3d)|
|-|-|

**Comment by ShonChassel** [comment]
hey 
Change it

<details open style="margin-left: 30px;">  
    <summary><h2><b> Guides </b></h2></summary>

- Here is [1st year guide](https://github.com/Imtjl/1st-year-guide) of the ITMO University Computer Science bachelor's program.

</details>

![image](https://github.com/community/community/assets/114100392/d2ffd5c8-55c5-4c9c-87ee-45d2f0ea966c)

**Reply by worthant** [reply]
Sorry haven't noticed your reply.
What i need to change it to?

Can you showcase, please, what you mean :)

**Reply by ShonChassel** [reply]
like in the pic 
put it like this 
![image](https://github.com/community/community/assets/114100392/ebdd888f-a084-4f83-b67a-5cc95b8eb428)

**Comment by ShonChassel** [comment]
?

**Reply by worthant** [reply]
As i said, nothing really changes, because github doesn't support 99% of the styles. 
As far as i am concerned, you can only align center/left/right and that's all that you can do
![image](https://github.com/community/community/assets/43885024/2f5e4793-5498-485c-8763-bad74ddce3dd)

**Comment by Wolanet** [comment]
Link anchors and headers seem bugged out for me too. Each header, either # or <h1/2etc.> was working fine until like 3 days ago, now they show up as a link (blue color too) in my README. Before you could only click on that "staplers" icon to being redirected to that header's paragraph, now it's like you have a full link to it.

It looks terrible and kinda wishing they'd fix this, if it was a bug or whatever.

**Comment by Wolanet** [comment]
> ?

why are you giving advice that doesn't work/makes no sense. At least try the code yourself first before suggesting other people to use it.

**Comment by Wolanet** [comment]
Check here, your issue might be related to this and the team at GitHub is investigating it. Supposedly.
https://github.com/orgs/community/discussions/67960#discussioncomment-7084352

edit: this issue seems to be finally fixed!
